packages/kbn-expect/expect.js

```diff
@@ -930,6 +935,10 @@ expect.eql = function eql(actual, expected) {
   // to determine equivalence.
   } else if (isRegExp(actual) && isRegExp(expected)) {
     return regExpEquiv(actual, expected);
+  // If both are Sets, they should be treated equal if the have the same
```

Comment:
spelling: `if they have the same`

Reply:
Thanks for catching it! Fixed in adf3b39.